ui/src/wide_article_preview.tsx

```diff
@@ -0,0 +1,154 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Model, S } from 'h2o-wave'
+import React from 'react'
+import { stylesheet } from 'typestyle'
+import { Component, XComponents } from './form'
+import { cards } from './layout'
+import { clas, cssVar, important, px } from './theme'
+import { bond, wave } from './ui'
+import { XPersona } from './persona'
+
+const css = stylesheet({
+  card: {
+    display: 'flex',
+    flexDirection: 'column',
+    padding: 24,
+  },
+  clickable: {
+    cursor: 'pointer'
+  },
+  header: {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'flex-end',
+    marginBottom: 24,
+    $nest: {
+      '.ms-Persona-primaryText': {
+        fontSize: 16,
+        lineHeight: px(22),
+        fontWeight: 500,
+      },
+      '.ms-Persona-secondaryText': {
+        fontSize: 14,
+        lineHeight: px(20),
+        color: cssVar('$text7')
+      },
+      '.ms-Persona-image': {
+        height: important('40px'),
+        width: '40px !important',
+      },
+      '.ms-Persona-imageArea': {
+        height: '40px !important',
+        width: '40px !important',
+      }
+    }
+  },
+  aux_value: {
+    marginBottom: 4,
+  },
+  content: {
+    flex: 1,
+    display: 'flex',
+  },
+  img: {
+    marginRight: 24,
+    flex: 1,
+    backgroundSize: 'cover',
+    backgroundRepeat: 'no-repeat',
+    backgroundPosition: 'center',
+  },
+  rhs: {
+    flex: 1,
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+  },
+  items: {
+    $nest: {
+      // Target mini button's icons and labels
+      '& .ms-Icon, & .ms-Button-label': {
+        color: cssVar('$text'),
+      }
+    }
+  },
+  title: {
+    marginBottom: 8,
+    lineHeight: 0.6,
+  },
+  caption: {
+    marginBottom: 24,
```

Comment:
This needs to be better thought through. Think of cases when there are no `items` specified. The code above would result in extra unwanted whitespace. The same goes the other way around - if `caption` is not specified, but `items` are.

Reply:
I added the logic to conditionally add space to `caption` based on whether `items` was specified. `title` is a required prop so I didn't have to handle the case where only `items` is provided on the right-hand side.